.github/workflows/build.yaml

```diff
@@ -112,3 +112,22 @@ jobs:
             --helm-extra-args '--timeout 60s' \
             --helm-extra-set-args '--set=image.tag=${{ needs.build.outputs.releaseTag }}' \
             --target-branch ${{ github.event.repository.default_branch }}
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.10.0
+
+      - name: Integration tests
+        run: |
+          pip install -r requirements.txt
+          pip install -r requirements-dev.txt
+          pytest -m integration
```

Comment:
Not sure if this should be its own job or if it should be folded into build flow?